packages/contracts/.gitignore

```diff
@@ -6,6 +6,8 @@ artifacts/
 abi/
 types/
 broadcast/
+.idea/
+.DS_Store
```

Comment:
you should use the root `.gitignore`, this one only affects the `contracts` folder.
and you should also remove the files themselves